### PR TITLE
feat: add backward compatibility for autoware_auto_msgs

### PR DESCRIPTION
**Pull request by kminoda**
## Description
Address https://github.com/orgs/autowarefoundation/discussions/3862

## How to review
Confirm that the existing test stacks work.

## Reference

https://github.com/orgs/autowarefoundation/discussions/3862

### CODE REVIEW

**Comment by kminoda** [comment]
@ktro2828 Hi, would you review this PR just in case?

File: perception_dataset/rosbag2/autoware_msgs.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
-)
+try:
+    from autoware_auto_perception_msgs.msg import (
+        DetectedObject,
+        DetectedObjects,
+        ObjectClassification,
+        TrackedObject,
+        TrackedObjects,
+    )
+except ModuleNotFoundError:
```
Comment (ktro2828) [inline perception_dataset/rosbag2/autoware_msgs.py:12]: [NITS] How about display the error message?

**Review by ktro2828 — APPROVED** [review]
The code almost looks good to me.

File: perception_dataset/rosbag2/autoware_msgs.py (hunk 1 above)
Comment (kminoda) [reply perception_dataset/rosbag2/autoware_msgs.py:14]: Thanks. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/104/commits/c22614c1977f4a991e61f36a7aaeda9f19a6d280

**Pull request merged**